hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7959,7 +7959,9 @@ public Result append(Append mutate, long nonceGroup, long nonce) throws IOExcept
         for (Map.Entry<Store, List<Cell>> entry: removedCellsForMemStore.entrySet()) {
           entry.getKey().add(entry.getValue());
         }
-        if (we != null) mvcc.complete(we);
```

Comment:
Changes to this file are a carrying forward of formatting changes from the branch-1.3 changes proposed on #275 and #277. They don't address a findbugs issue, only try to keep the code as consistent as possible between branches.